devel/gettext/files/patch-libtextstyle-libproc.diff

```diff
@@ -1,8 +1,8 @@
 Combination of:
 https://github.com/coreutils/gnulib/commit/8e3d5944a4c6fca1ecf1d76f669da83861eb0ca0
 https://github.com/coreutils/gnulib/commit/119622a83d47b01b5a9fb2af4542cdb45f4eb83b
---- lib/get_ppid_of.c.orig
-+++ lib/get_ppid_of.c
+--- libtextstyle/lib/get_ppid_of.c.orig
++++ libtextstyle/lib/get_ppid_of.c
```

Comment:
Are you certain that this should be changed? This patch was added by @evanmiller to fix build failure on Tiger. I would want confirmation from Evan before making this change.

Reply:
@ryandesign yes, because before patch were applied to subport and `patch.dir` was adjusted.